src/test/test.ts

```diff
@@ -0,0 +1,34 @@
+export const defaultAnsibleConfigurations = {
```

Comment:
Why is this duplicate declaration needed?

Reply:
Same here, `test.ts` and `test.js` are not actually the part of the PR. I will remove it. Thanks.